### DISCUSSION TITLE
OpenStreetMap attribution is missing

### DISCUSSION THREAD

**Discussion by habi** [transitland/transitland#341] [Q&A] [answered]
As an OpenStreetMap contributor, I am pleased to see that you are using OpenStreetMap on your website: https://www.transit.land/map/

However, I noticed that the map seems to lack the proper credit, which you either deliberately removed if you're using a tile provider or did not add if you're providing your own tiles.
OpenStreetMap's license requires that you both credit OpenStreetMap and make users aware of the data license (by linking to our copyright page).

For more information, see: https://www.openstreetmap.org/copyright

The correct attribution helps to avoid misunderstandings about the source of the map or data, as well as what others are allowed to do with it.
If you need help making the changes or have further questions, feel free to ask question in reply to this post or in one of 
OpenStreetMap's communication channels: https://wiki.openstreetmap.org/wiki/Contact

More importantly attribution helps us spread the word about OpenStreetMap and attract new contributors and build the community.
This is incredibly important to improving the quality of the database that you and millions of others rely on.

**Comment by drewda** [comment, marked as answer]
Hello. We do have an attribution line and link for OpenStreetMap attribution (along with other data sources) on https://www.transit.land/map/ However, it appears that in some browsers this is being rendered further down the page, requiring the user to scroll:

<img width="1588" alt="image" src="https://user-images.githubusercontent.com/212369/139725326-3e0ee73f-d711-4a27-b00c-a5bb39dfca63.png">

We'll see if we can fix this styling so that the attribution line appears in its intended place in all browsers.

**Reply by drewda** [reply]
We've now fixed the styling, so the attribution line should appear in its intended place:

<img width="1258" alt="image" src="https://user-images.githubusercontent.com/212369/139782889-c0e318c3-cf9d-40a9-85c4-06c76abf01a1.png">

Thanks for bringing this to our attention.

**Reply by habi** [reply]
Awesome, thanks for the quick fix!
